packages/apollo-gateway/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ### vNEXT
 
+- fix @requires bug preventing array and null values [PR #2928](https://github.com/apollographql/apollo-server/pull/2928)
```

Comment:
I don't know where our users are expecting the `CHANGELOG.md` entries for this to appear, but we're back and forth between the root `CHANGELOG.md` right now.  The root `CHANGELOG.md` certainly doesn't make a lot of sense either though, but we're going to need tooling to get this right.

At the very least though, here's my suggestion:

```suggestion
- Fix `@requires` bug preventing array and null values. [PR #2928](https://github.com/apollographql/apollo-server/pull/2928)
```

Mainly to avoid linking to the user known as `requires`, https://github.com/requires.